src/core/services/utility.spec.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+import {isObject} from './utility';
+
+describe('utility', () => {
+
+	describe('isObject', () => {
+
+		it('should return false when passing number', () => {
+			assert.isFalse(isObject(12));
```

Comment:
I like bdd-style more. Especially it can be seen for complex cases:
`expect(12).not.to.be.an.instanceof(Object)`
I think, in this case we'll use all power of Chai